src/app/components/ms-item/ms-item.component.scss

```diff
@@ -0,0 +1,22 @@
+.section {
+    margin-bottom: 20px;
+}
+
+.col-1 {
+    max-width: 7%;
+}
+
+.has-nested, .has-nested2 {
+    font-size: 0.95rem;
+    .col-label {
+        max-width: 4%;
```

Comment:
Avoid using the percentage value if it's not necessary. Bootstrap has default utility classes the might be usefull in this cases https://getbootstrap.com/docs/4.1/utilities/spacing/
Use boostrap grid!